screens/Contacts/ContactGraph/ContactGraph.tsx

```diff
@@ -59,8 +58,8 @@ const ContactGraph = ({
     // Dismiss the keyboard when the user taps outside of the search bar
     <TouchableWithoutFeedback onPress={() => Keyboard.dismiss()}>
       <View style={styles.container}>
-        <TextInput
-          style={[globalStyles.text, styles.searchBar]}
+        <InputField
+          label=""
```

Comment:
why do you add an empty label? What purpose does it serve?

Reply:
The InputField needed it but I made the label prop optional, that's definitely better.